app/src/main/java/com/quran/labs/androidquran/ui/PagerActivity.java

```diff
@@ -2132,4 +2140,14 @@ public void onPanelExpanded(View panel) {
     public void onPanelAnchored(View panel) {
     }
   }
+
+  private class NoisyAudioStreamReceiver extends BroadcastReceiver {
+    @Override
+    public void onReceive(Context context, Intent intent) {
+      if (AudioManager.ACTION_AUDIO_BECOMING_NOISY.equals(intent.getAction())) {
```

Comment:
@ozbek I just added an extra check to see if the current mode is playing before pausing.

I guess what you suggested could be more efficient, but I'll have to modify the unregister logic to ensure it only happens under certain conditions.

What do you think?